.run_docker_tests.sh

```diff
@@ -16,6 +16,9 @@ uname -m
 echo "Output of sys.maxsize in Python:"
 python -c 'import sys; print(sys.maxsize)'
 
+# temporary
```

Comment:
Maybe specify the conditions of temporary?  Just so that someone else might know when to remove it...?